### PR TITLE
Action Page Sponsor Updates

### LINKED ISSUE #706: action page (do it) – photo gallery styles
The Polaroid-style image need styling.

### LINKED ISSUE #709: action page (prove it) – rules and regulations
The "Rules and Regulations" link should appear when uploaded to the campaign node.

### LINKED ISSUE #1041: action page – sponsor header colors
Sponsored campaigns should have custom header colors to reflect the sponsor's branding.

Requires  #895

### LINKED ISSUE #1042: action page (prove it) – sponsor background pattern
Sponsored campaigns will have the option of a custom background pattern.

### LINKED ISSUE #1043: action page (header) – hero image update
Now that the crop sizes have been decided, a `16:9` landscape image should be shown for "desktop" views and a `1:1` square image should be shown for "tablet" and below.

### LINKED ISSUE #1095: action page (header) – default hero images
If hero images are not provided on the campaign node a default image should load. Fantini requested that we use H&M hero images as the defaults for now.

### PR DESCRIPTION
**Pull request by mmwtsn**
Adds:
- alternate header colors to fix #1041
- alternate "prove it" pattern to fix #1042
- alternate hero image to fix #1043
- default hero image to fix #1095
- polaroid "do it" image to fix #706
- rules and regulations to fix #709

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign--pitch.tpl.php [hunk 1]
```diff
@@ -1,29 +1,32 @@
 <section class="campaign--wrapper">
-  <header class="header <?php print $classes; ?>" <?php print (isset($hero_img_l_url) ? 'style="background:url(' . $hero_img_l_url . ');"' : ''); ?>>
-    <div class="meta">
-      <h1 class="title"><?php print $title; ?></h1>
-      <p class="cta"><?php print $cta; ?></p>
 
-      <?php if (isset($end_date)): ?><p class="date"><?php print $end_date; ?></p><?php endif; ?>
+  <?php foreach ($hero_image as $image_url ) :?>
```
Comment (aaronschachter) [inline lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign--pitch.tpl.php]: Doesn't this need the same `$hero_image as $key => $image_url` ?  Don't think `$key` will be defined below

File: lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module [hunk 2]
```diff
@@ -334,15 +334,19 @@ function dosomething_campaign_preprocess_node(&$vars) {
...
       $hero_nid = $vars['field_image_campaign_cover'][0]['entity']->nid;
       $vars['hero_image_l'] = dosomething_image_get_themed_image($hero_nid, 'landscape', '1920x1280p');
-      $vars['hero_image_m'] = dosomething_image_get_themed_image($hero_nid, 'portrait', '640x800');
+      $vars['hero_image_m'] = dosomething_image_get_themed_image($hero_nid, 'square', '300x300');
       $vars['hero_image_l_url'] = dosomething_image_get_themed_image_url($hero_nid, 'landscape', '1920x1280p');
-      $vars['hero_image_m_url'] = dosomething_image_get_themed_image_url($hero_nid, 'landscape', '640x800');
+      $vars['hero_image_m_url'] = dosomething_image_get_themed_image_url($hero_nid, 'square', '300x300');
     }
 
+    // Fall back to default values if a hero image was not set.
```
Comment (barryclark) [inline lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module:347]: Maybe an @TODO here to move the images from Trello to the repository?

Comment (mmwtsn) [inline lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module:347]: Searching "trello" will pull those up. I think that's OK.

File: lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign--pitch.tpl.php (hunk 1 above)
Comment (mmwtsn) [inline lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign--pitch.tpl.php]: Ah, good catch. The two templates are out of sync. Thanks.

**Comment by barryclark** [comment]
+1 :bangbang:

**Comment by aaronschachter** [comment]
Looks good here too - ship it

**Pull request merged**